.github/workflows/python.yml

```diff
@@ -0,0 +1,198 @@
+name: Python
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - '*'
+    paths:
+      - python/**
+  pull_request:
+    paths:
+      - python/**
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: "3.8"
+      - name: Build
+        uses: PyO3/maturin-action@v1
+        with:
+          command: develop
```

Comment:
I see that CI is failing on this develop command.

People have different opinions on whether you should use a lockfile for running tests or not. I tend to prefer it, to make sure 

Anyways, you can see how I use poetry as the venv for maturin here, if you want: https://github.com/kylebarron/arro3/blob/0829e34fe250314c2e068ff86e3c5e7ad003d607/.github/workflows/test-python.yml#L56-L91